pkg/operator/starter.go

```diff
@@ -135,6 +160,43 @@ func RunOperator(ctx *controllercmd.ControllerContext) error {
 		ctx.EventRecorder,
 	)
 
+	nodeProvider := DaemonSetNodeProvider{
+		TargetNamespaceDaemonSetInformer: kubeInformersForNamespaces.InformersFor(operatorclient.TargetNamespace).Apps().V1().DaemonSets(),
+		NodeInformer:                     kubeInformersForNamespaces.InformersFor("").Core().V1().Nodes(),
+	}
+	deployer, err := encryptiondeployer.NewRevisionLabelPodDeployer("revision", operatorclient.TargetNamespace, kubeInformersForNamespaces, resourceSyncController, kubeClient.CoreV1(), kubeClient.CoreV1(), nodeProvider)
+	if err != nil {
+		return err
+	}
+
+	encryptionControllers, err := encryption.NewControllers(
```

Comment:
should we change `NewControllers` to accept an alternative implementation of the `prune` ctrl?

Reply:
Not sure we should couple the prune controller with the encryption controllers without a need.